extensions/postgres/schema/xdb_sys_schema.sql

```diff
@@ -16,7 +16,7 @@ CREATE TABLE xdb_sys_process_executions(
     timeout_seconds INTEGER,
     history_event_id_sequence INTEGER,
     state_execution_sequence_maps jsonb NOT NULL , -- some maps from stateId and sequence number
-    info jsonb , -- workerURL, processType, etc
+    info jsonb , -- workerURL, processType, waitToComplete, etc
```

Comment:
It will be slightly better perf to use a separate bool field, as updating a blob will be more cost. 
so usually, I would prefer using this info field to store static info only. (And that’s why other info like sequenceMap is in separate field) 

However , not that this also because of this stage of the project. This blob field can be used to add new fields without schema change. After we make a production release, we may choose to use it for adding even a bool field so that users don’t have to change schema in order to upgrade

Reply:
Learnt!